rsconf/package_data/rsconf/rsconf.sh

```diff
@@ -264,13 +264,16 @@ rsconf_install_perl_rpm() {
     # both fresh install and update, which install does, but
     # it doesn't return an error if the update isn't done.
     # You just have to check so this way is more robust
-    declare reinstall=
+    declare install_cmd=
```

Comment:
Nice change. This is a local variable so we tend to use single letters. (I know this is not always the case in rsconf.sh, but it's older than the single letter variable policy.) This function is long so it's probably fine to leave as is. Bash is harder, because function syntax in bash is annoying.

Reply:
I'll leave this as there are a few other locals that would need to be updated, and it's a longer function as you say.

Reply:
It's a tricky thing with shell functions, since vars are also dynamically scoped...